app/javascript/components/patient/Patient.js

```diff
@@ -724,7 +724,7 @@ class Patient extends React.Component {
             </Row>
           </div>
         </Collapse>
-        {this.props.display_follow_up_flag && (
+        {this.props.can_modify_subject_status && (
```

Comment:
so here I think you can just do a check for `can_modify_subject_status` and `edit_mode`.  then you can remove the additional prop in `Review.js` (see next comment)